### PR TITLE
SWE updates to be compliant with new cdflib releases

### PR DESCRIPTION
**Pull request by tech3371**
# Change Summary
main changes:
* updated SWE code to use use datetime64 for Epoch variable and added fieldname and description missing for science data.
* Updated cdflib version
* Update tests to match above changes

### CODE REVIEW

**Review by bourque — APPROVED** [review]
Looks good!

File: imap_processing/swe/l1a/swe_science.py [hunk 1]
```diff
@@ -169,16 +173,30 @@ def swe_science(decom_data):
...
-        attrs=swe_cdf_attrs.l1a_science_attrs.output(),
+        attrs=science_attrs.output(),
     )
 
+    raw_science_attrs = dataclasses.replace(
+        swe_cdf_attrs.l1a_science_attrs,
+        catdesc="Raw counts from SWE",
+        fieldname="Raw counts from SWE",
+        label_axis="Raw counts from SWE",
+        units="counts",
```
Comment (bryan-harter) [inline imap_processing/swe/l1a/swe_science.py:194]: I don't know how much I should be focusing on ISTP stuff at the moment, (my guess is not much?) but for the record I'd probably have:

catdesc="Raw electron counts at 12 energy bins between (xx) and (yy) eV"
fieldname="Raw Electron Counts"
label_axis="Counts"

Comment (bryan-harter) [reply imap_processing/swe/l1a/swe_science.py:194]: Whoops you merged right as I was writing lol

Comment (tech3371) [reply imap_processing/swe/l1a/swe_science.py:194]: sorry Bryan! Please add comment and I will open with new changes.

**Pull request merged**